app/src/main/java/com/gh4a/adapter/CodeSearchAdapter.java

```diff
@@ -55,28 +59,48 @@ protected void onBindViewHolder(ViewHolder holder, CodeSearchResult result) {
                         int start = item.getStartPos();
                         int end = item.getEdndPos();
```

Comment:
Ohh I see now. Also, I'm not sure if you are using the app right now, but if you do then replying looks to be broken: 
![spectacle x14496](https://cloud.githubusercontent.com/assets/5156340/26406358/5ca07b2a-4098-11e7-8dae-dd036040769e.png)